packs/nest/actions/fan.yaml

```diff
@@ -0,0 +1,24 @@
+---
+name: fan
+runner_type: run-python
+description: Manage fan state
+enabled: true
+entry_point: fan.py
+parameters:
+  state:
+    type: string
+    description: State to set the fan
+    enum:
+      - "on"
```

Comment:
I prefer explicit quoting. I have seen some weird parse errors if the value is unquoted.